src/explore-education-statistics-common/src/modules/find-statistics/components/charts/VerticalBarBlock.tsx

```diff
@@ -18,31 +20,31 @@ export class VerticalBarBlock extends Component<ChartProps> {
     });
 
     return (
-      <BarChart
-        width={900}
-        height={this.props.height || 300}
-        data={chartData}
-        margin={{ top: 5, right: 30, left: 20, bottom: 25 }}
-      >
-        <XAxis
-          dataKey={this.props.xAxis.key || 'name'}
-          interval={0}
-          tick={{ fontSize: 12 }}
-        />
-        <CartesianGrid />
-        <YAxis />
-        <Tooltip />
-        <Legend />
-
-        {this.props.chartDataKeys.map((key, index) => (
-          <Bar
-            key={index}
-            dataKey={key}
-            fill={colours[index]}
-            name={this.props.labels[key]}
+      <ResponsiveContainer width={900} height={this.props.height || 300}>
+        <BarChart
+          data={chartData}
+          margin={{ top: 5, right: 30, left: 20, bottom: 25 }}
+        >
+          <XAxis
+            dataKey={this.props.xAxis.key || 'name'}
+            interval={0}
+            tick={{ fontSize: 12 }}
           />
-        ))}
-      </BarChart>
+          <CartesianGrid />
+          <YAxis />
+          <Tooltip />
+          <Legend />
+
+          {this.props.chartDataKeys.map((key, index) => (
+            <Bar
+              key={index}
```

Comment:
Similarly, to previously, feels like we should be using `key` as the key instead of index.

Reply:
as before